dashboard/db/migrate/20240807174943_create_aichat_requests.rb

```diff
@@ -0,0 +1,15 @@
+class CreateAichatRequests < ActiveRecord::Migration[6.1]
+  def change
+    # use utf8mb4 to support emoji
+    create_table :aichat_requests, charset: 'utf8mb4', collation: 'utf8mb4_bin' do |t|
+      t.integer :user_id, null: false
+      t.json :model_customizations, null: false
+      t.json :stored_messages
+      t.text :new_message, null: false
```

Comment:
In `aichat_controller`, 'newMessage' is a `ChatMessage` with fields `role`, `chatMessageText`, and `status`. Are we only storing the message text in this table? 

If we are storing other fields (`role` and `status`): Would it be helpful to store as json as well? And maybe rename `status` as `request_status` below?

Reply:
Oh good catch! yeah this should probably be `json`

Reply:
Also, discussed offline - will rename `status` to `execution_status`

Reply:
Thanks for the name update!